dp/_visualizer.py

```diff
@@ -126,7 +126,7 @@ def __init__(self):
 
         # https://dash-bootstrap-components.opensource.faculty.ai/docs/themes/
         # If we use a dark theme, make the layout background transparent
-        themes = [dbc.themes.LUX]
+        themes = [dbc.themes.JOURNAL]
```

Comment:
See comment in conversation, but this makes the math in the sidebar overflow the sidebar.

Reply:
This is also a problem with different monitor resolutions. On my end it doesn't overflow. So if a user opens the page with a small monitor it will always overflow.